BrickBrackBruck/Ding.cs

```diff
@@ -0,0 +1,39 @@
+
+using System.Collections.Generic;
+using System.Collections;
+
+class Ding
+    {
+        public string bezeichnung;
+
+        public ArrayList werte = new ArrayList();
+        public ArrayList änderungen = new ArrayList();
+        public ArrayList volumina = new ArrayList();
+        
+        // statistisches zeugs
+        public int gültigeWerte = 0;
+        public int kleinsterIndex = -1;
```

Comment:
Ist mir latte. Dann initialisiere ich halt nicht. Alte Gewohnheit. Ein Ding kann eh nur in der Funktion "DingHinzufügen" erzeugt werden. Und dort werden die Werte immer gesetzt.